homeassistant/components/ombi/sensor.py

```diff
@@ -0,0 +1,72 @@
+"""Support for Ombi."""
+import logging
+
+from pyombi import OmbiError
+
+from homeassistant.helpers.entity import Entity
+
+from .const import DOMAIN, SENSOR_TYPES
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def setup_platform(hass, config, add_entities, discovery_info=None):
+    """Set up the Ombi sensor platform."""
+    sensors = []
```

Comment:
Please add a guard clause here that checks if discovery_info is None and return if so. This will guard from setting up the platform via a platform config section.